<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBgProp</class>
 <widget class="QDialog" name="HBgProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>背景设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,4,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>背景设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>背景颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bgBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>网格颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gridBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>元件颜色</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>填充颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fillBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>管脚颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lineBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>前景颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="upedgeBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>后景颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="downedgeBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>阴影颜色:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shadowBtn">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="defaultBtn">
       <property name="text">
        <string>默认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setBtn">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
